ledger/acctupdates.go

```diff
@@ -739,6 +746,17 @@ func (au *accountUpdates) onlineTotalsImpl(rnd basics.Round) (basics.MicroAlgos,
 	return totals.Online.Money, nil
 }
 
+// totalsImpl returns the online totals of all accounts at the end of round rnd.
+func (au *accountUpdates) totalsImpl(rnd basics.Round) (ledgercore.AccountTotals, error) {
```

Comment:
Perhaps we do this a lot, but "Impl" always bugs be as a name because it doesn't say anything about why it's pulled out.  Here I guess the point is that it is `totalsWithoutLocking` or some such. (I'd love a more concise convention for such cases.)

Here, since it's only called from Totals, I guess I'd rather just inline it.